src/wovnio/wovnphp/API.php

```diff
@@ -91,6 +93,22 @@ public static function translate($store, $headers, $original_content)
         }
     }
 
+    private static function getUriRepresentation($uri, $store, $lang)
+    {
+        $urlPatternName = $store->settings['url_pattern_name'];
+
+        if ($urlPatternName == 'custom_domain') {
+            $customDomainLangs = $store->getCustomDomainLangs();
```

Comment:
When is this null? Ideally this would be validated during startup, but it seems we don't do any settings validation at all.

Reply:
Not suggesting any changes though. It's just a bit annoying to need to check for null everywhere.

Reply:
I guess it's never null unless the constructor for CustomDomainLangs have some fatal error? And it's true that it's a big problem in WOVN.php that we don't do validations.

Reply:
It will be null if they use custom domain pattern but don't define the custom_domain_langs config. In which case they have bigger problems. In other backends we try to validate the settings and refuse to start.

Reply:
Making PHP refuse to run is very difficult...